build_openstudio_gems.rb

```diff
@@ -155,17 +156,38 @@ def make_package(install_dir, tar_exe, expected_ruby_version)
     end
   end
 
-  Dir.chdir("#{install_dir}/..")
-
   new_file_name = "openstudio3-gems-#{DateTime.now.strftime("%Y%m%d")}-#{platform_prefix}.tar.gz"
+  File.open("#{install_dir}/version.txt", 'w') do |f|
+    f.puts new_file_name
+  end
```

Comment:
Write the tar.gz name into a text file at the root of the extracted dir, so we can get it from CMake to ensure we always use the right version when switching branches: checking that we have the .tar.gz locally and it has the right MD5sum doesn't suffice to tell that the extracted directory is the right now, as you may have multiple openstudio3-gems tar.gzs.